types/twit/index.d.ts

```diff
@@ -310,6 +310,10 @@ declare module 'twit' {
             in_reply_to_status_id?: number | string;
             page?: number;
             auto_populate_reply_metadata?: boolean;
+            list_id?: number | string;
+            name?: string;
+            description?: string;
+            mode?: string;
```

Comment:
It looks like `mode` can have two possible values. Might as well specify those.

```suggestion
            mode?: 'public' | 'private';
```